example/index.js

```diff
@@ -1 +1 @@
+!function(e){if("object"==typeof exports&&"undefined"!=typeof module)module.exports=e();else if("function"==typeof define&&define.amd)define([],e);else{("undefined"!=typeof window?window:"undefined"!=typeof global?global:"undefined"!=typeof self?self:this).copyToClipboard=e()}}(function(){return function a(c,l,i){function s(t,e){if(!l[t]){if(!c[t]){var o="function"==typeof require&&require;if(!e&&o)return o(t,!0);if(u)return u(t,!0);var n=new Error("Cannot find module '"+t+"'");throw n.code="MODULE_NOT_FOUND",n}var r=l[t]={exports:{}};c[t][0].call(r.exports,function(e){return s(c[t][1][e]||e)},r,r.exports,a,c,l,i)}return l[t].exports}for(var u="function"==typeof require&&require,e=0;e<i.length;e++)s(i[e]);return s}({1:[function(e,t,o){"use strict";var u=e("toggle-selection");t.exports=function(t,o){var n,r,e,a,c,l,i,s=!1;o||(o={}),o.style&&(i=Object.assign({background:"white",text:"black"},o.style)),n=o.debug||!1;try{if(e=u(),a=document.createRange(),c=document.getSelection(),(l=document.createElement("span")).textContent=t,l.style.all="unset",i&&(l.style.backgroundColor=i.background,l.style.color=i.text),l.style.position="fixed",l.style.top=0,l.style.clip="rect(0, 0, 0, 0)",l.style.whiteSpace="pre",l.style.webkitUserSelect="text",l.style.MozUserSelect="text",l.style.msUserSelect="text",l.style.userSelect="text",l.addEventListener("copy",function(e){e.stopPropagation(),i||(e.preventDefault(),e.clipboardData.clearData(),e.clipboardData.setData("text/plain",t))}),document.body.appendChild(l),a.selectNodeContents(l),c.addRange(a),!document.execCommand("copy"))throw new Error("copy command was unsuccessful");s=!0}catch(e){n&&console.error("unable to copy using execCommand: ",e),n&&console.warn("trying IE specific stuff");try{window.clipboardData.setData(o.format||"text",t),s=!0}catch(e){n&&console.error("unable to copy using clipboardData: ",e),n&&console.error("falling back to prompt"),r=function(e){var t=(/mac os x/i.test(navigator.userAgent)?"⌘":"Ctrl")+"+C";return e.replace(/#{\s*key\s*}/g,t)}("message"in o?o.message:"Copy to clipboard: #{key}, Enter"),window.prompt(r,t)}}finally{c&&("function"==typeof c.removeRange?c.removeRange(a):c.removeAllRanges()),l&&document.body.removeChild(l),e()}return s}},{"toggle-selection":2}],2:[function(e,t,o){t.exports=function(){var t=document.getSelection();if(!t.rangeCount)return function(){};for(var e=document.activeElement,o=[],n=0;n<t.rangeCount;n++)o.push(t.getRangeAt(n));switch(e.tagName.toUpperCase()){case"INPUT":case"TEXTAREA":e.blur();break;default:e=null}return t.removeAllRanges(),function(){"Caret"===t.type&&t.removeAllRanges(),t.rangeCount||o.forEach(function(e){t.addRange(e)}),e&&e.focus()}}},{}]},{},[1])(1)});
```

Comment:
This is the result from running `yarn pretest`, so that I could test using the example code.